devsite/docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+version: '3.5'
+services:
+
```

Comment:
do you plan to add Figures here as well?

Reply:
@OmarIthawi TBD